package.json

```diff
@@ -97,7 +97,7 @@
     "cross-env": "4.0.0",
     "cross-spawn": "5.0.1",
     "css-loader": "0.28.0",
-    "cypress": "3.4.1",
+    "cypress": "12.8.1",
```

Comment:
@kikuomax I assume there would be some issues with the e2e tests after upgrading cypress to the latest version.

Reply:
E2e tests are not functioning, in effect, anyway.